sdk/tests/conformance2/extensions/ext-texture-norm16.html

```diff
@@ -153,6 +153,23 @@
                                 "framebuffer should not be complete with SNORM16 texture attached");
 }
 
+function runInternalFormatQueryTest()
+{
+    debug("");
+    debug("testing the internal format query");
+
+    var maxSamples = gl.getParameter(gl.MAX_SAMPLES);
+    var formats = new Array(ext.R16_EXT, ext.RG16_EXT, ext.RGBA16_EXT);
```

Comment:
Was this copied from somewhere?
`const formats = [ext.R16_EXT, ext.RG16_EXT, ext.RGBA16_EXT];`

Reply:
Yes. Fixed the copy src with the same style